html5ever/tests/serializer.rs

```diff
@@ -165,15 +165,15 @@ test!(noframes_literal, r#"<noframes>(x & 1) < 2; y > "foo" + 'bar'</noframes>"#
 
 test!(pre_lf_0, "<pre>foo bar</pre>");
 test!(pre_lf_1, "<pre>\nfoo bar</pre>", "<pre>foo bar</pre>");
-test!(pre_lf_2, "<pre>\n\nfoo bar</pre>");
+test!(pre_lf_2, "<pre>foo bar</pre>");
```

Comment:
oh whoops.. my mistake.. not sure what was going on in my mind :|

---

Should it actually be `test!(pre_lf_2, "<pre>\n\nfoo bar</pre>", "<pre>\nfoo bar</pre>");`? 

> For example, consider the following markup:
>``` 
><pre>
>
>Hello.</pre>
>```
>
>When this document is first parsed, the pre element's child text content starts with a single newline character. After a serialize-reparse roundtrip, the pre element's child text content is simply "Hello.".

I'm don't understand what's meant by a serialize roundtrip.. but `test!(pre_lf_2, "<pre>\n\nfoo bar</pre>", "<pre>foo bar</pre>");` doesn't seem to be right